internal/pkg/cli/task_run.go

```diff
@@ -0,0 +1,270 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"errors"
+	"fmt"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/config"
+	"github.com/aws/amazon-ecs-cli-v2/internal/pkg/docker/dockerfile"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"regexp"
+	"strings"
+)
+
+var regions = []string{
```

Comment:
I'm not sure having a hard coded list of regions is ideal here as a new region being spun up will cause this to fail until we can update the list.